### DISCUSSION TITLE
How to use pseudo-elements and pseudo-classes easily

### DISCUSSION THREAD

**Discussion by studyforhuihui** [unocss/unocss#3611] [Q&A] [answered]
When I need to write long pseudo-elements or pseudo-class attributes, I have to add signs in front of them: such as:hover,:::before, etc. I feel it will be very troublesome. Can I use curly brackets or something like that? They are grouped together so that I only have to write one prefix

**Comment by Ark21bit** [comment, marked as answer]
Yes it is possible, https://unocss.dev/transformers/variant-group

**Reply by studyforhuihui** [reply]
Thank you so much!It solves a big problem for me
